include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -158,3 +158,19 @@ static const char* const kOrtSessionOptionsConfigForceSpinningStop = "session.fo
 // "0": in some cases warnings will be logged but processing will continue. The default.
 // May be useful to expose bugs in models.
 static const char* const kOrtSessionOptionsConfigStrictShapeTypeInference = "session.strict_shape_type_inference";
+
+// This Option allows setting affinities for intra op threads.
```

Comment:
I was wondering if we need both the C API, where we have lots of them already and the option?